build.gradle

```diff
@@ -30,8 +30,3 @@ allprojects {
         }
     }
 }
-
```

Comment:
Don't delete this, it's used to upgrade gradle.

Reply:
Overriding wrapper scripts is deprecated and will be removed in future gradle versions

Reply:
@gengjiawen looks like there it is (hope) https://docs.gradle.org/4.8/release-notes.html#overwriting-gradle's-built-in-tasks